src/nss/seed_treasure.nss

```diff
@@ -3,6 +3,36 @@
 #include "nwnx_item"
 #include "nwnx_util"
 
+
+int GetIsItemConsumableMisc(object oItem)
```

Comment:
This is a good way of handling consumables, though I wonder if it works for traps. I originally envisioned removing them from misc and giving it a `consume` resref. Tedious but more accurate.

Should traps even be counted as consumable? Not sure.

Reply:
You're right, I did forget to put traps in here!

Doing that would however make them much more common and possibly swamp the other consumables (adding the various nonconsumable misc sellable items has already them much rarer than before), and as items that can't be sold I don't know if giving out more of them is necessarily a good thing.

Then again I don't think I've ever used a trap kit in NWN in my life... so I'm maybe not the best to make the call.

Reply:
You're right on swamping the other items. Let's think about it later. The problem is that there are SO many trap types it might make it a lot more common than it should be. As it stands if we wanted to make changes to it it should probably be in it's own tier.